other/block-cluster-admin-from-ns/04-pods.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- command: kubectl --context=testuser-context create -f good-pod.yaml
```

Comment:
Also try a DELETE and UPDATE operation. Obviously, you'll want to create something first as the non `testuser` account, but it could be as simple as a ConfigMap.

Reply:
Done!